app/dedicated-cloud-gateways/transit-gateways.md

```diff
@@ -1,27 +1,141 @@
+I(<img src="/assets/logos/konglogo-gradient-secondary.svg" style="max-height:32px" class="no-image-expand"/>Konnect \n#40;fully-managed \ndata plane#41;)
+J(Internet)
+
+subgraph 1 [User AWS Cloud]
+    subgraph 2 [Region]
+        subgraph 3 [Virtual Private Cloud #40;VPC#41;]
+        A
+        B
+        C
+        end
+        A & B & C <--> D
+    end
+   D<-->E
+end
+
+subgraph 4 [Kong AWS Cloud]
+    subgraph 5 [Region]
+        E<-->F
+        F <--private API \n access--> G & H & I
+        subgraph 6 [Virtual Private Cloud #40;VPC#41;]
+        G
+        H
+        I
+        end
+    end
+end
+
+G & H & I <--public API \n access--> J
+
+
+{% endmermaid %}
+<!--vale on-->
+
+## AWS Configuration for Transit Gateway Peering
+
+This process includes three main steps: 
+
+1. Create and Share the Transit Gateway in AWS
+
+    * Navigate to **VPC > Transit Gateways** in the AWS Console.
+    * Select **Create transit gateway**, provide a name, and create the gateway.
+    * Save the **Transit Gateway ID**.
+    * Open the **Resource Access Manager**, and select **Create Resource Share**.
+    * Choose **Transit Gateways** as the resource type and select the newly created gateway.
+    * Name the resource share and retain default managed permission settings.
+    * Enable **Allow external accounts**, choose **AWS Account**, and enter the **AWS ID** from the {{site.konnect_short_name}} UI (**Gateway Manager > Networks**).
+    * Create the resource share and save the resulting **RAM Share ARN**.
+
+2. Accept the Transit Gateway Attachment in AWS
+
+    * Go to **VPC > Transit Gateway Attachments** in the AWS Console.
+    * Locate the incoming attachment request from the {{site.konnect_short_name}} AWS Account ID.
+    * Accept the request to establish the connection.
+
+    Each AWS VPC that needs to send or receive traffic must have its own Transit Gateway attachment.
+
+## Konnect Configuration for Transit Gateway Peering
+
+To finish setup in {{site.konnect_short_name}}:
+
+* Go to **Gateway Manager > Networks**.
+* Select your network and click **Attach Transit Gateway**.
+* Provide the following information:
+  * Transit Gateway Name
+  * One or more CIDR blocks (must not overlap with your {{site.konnect_short_name}} network)
+  * RAM Share ARN
+  * Transit Gateway ID
+
+## Accept Transit Gateway Attachment in AWS
+
+In the AWS Console:
+
+* Go to **VPC > Transit Gateway Attachments**
+* Wait for an attachment request from the Konnect AWS Account ID
+* Accept the request
+
+Ensure that each AWS VPC requiring traffic forwarding has its own Transit Gateway attachment.
+
+After the attachment is active, create a route in your AWS VPC to forward traffic to the {{site.konnect_short_name}} managed VPC through the Transit Gateway. This ensures proper traffic flow from {{site.konnect_short_name}} to your Services and back.
+
+## DNS Mappings
+
+Add IP addresses of DNS servers and the corresponding domains. Supported mapping types include:
```

Comment:
Where I add these addresses is unclear. Is this part of the CIDR block when I configure Konnect? If so, I'd make this an h3 below that heading and make it clear that this is how the mapping works when they configure that setting.